README.md

```diff
@@ -89,7 +89,7 @@ If you want to test it in your current setup, just use the modified launch files
 
 If you would like to run this package to connect to the hardware, you only need to run the following launch file.
 ```
-roslaunch ur_modern_driver urXX_bringup.launch robot_ip:=ROBOT_IP_ADDRESS
+roslaunch ur_modern_driver urXX_bringup.launch robot_ip:=ROBOT_IP_ADDRESS [reverse_port:=REVERSE_PORT]
```

Comment:
I'm not sure I'd add this here, since there's quite a lot of extra arguments in those launch files that are not described in the documentation.

Reply:
I added it here, since it is also stated in the [original driver's README](https://github.com/ros-industrial/universal_robot/blob/kinetic-devel/README.md). I am also fine with leaving it out or adding it differently...

Reply:
We have to review carefully the documentation, add add information about all supported parameters (related to https://github.com/ros-industrial/ur_modern_driver/issues/245#issuecomment-447817317)

I am agree with @miguelprada and, for now and until we create a proper documentation, do not parse the `reverse_port` as an argument for the launch file in the Readme